.github/workflows/release-embedding-sdk.yml

```diff
@@ -414,16 +409,21 @@ jobs:
       - name: Check out the code using the provided branch
         uses: actions/checkout@v4
         with:
+          token: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
           ref: ${{ inputs.branch }}
 
-      # Lightweight tags don't need committer name and email
+      - name: Setup git user
+        run: |
+          git config --global user.email "github-automation@metabase.com"
+          git config --global user.name "Metabase Automation"
+
       - name: Create and push SDK version tag
         run: |
-          git tag ${{ env.tag }}
+          git tag -a ${{ env.tag }} -m "Tagging SDK version ${{ env.tag }}"
           git push origin ${{ env.tag }}
 
       - name: Create and push SDK stable version tag
         if: ${{ startsWith(inputs.branch, 'release-x.') }}
         run: |
-          git tag -f ${{ env.stable_tag }}
+          git tag -a -f ${{ env.stable_tag }} -m "Tagging SDK stable version ${{ env.stable_tag }}"
```

Comment:
I didn't test this tag because we keep modifying it.
https://github.com/metabase/metabase/blob/8bd5d2e07ff0a273098cbc679b084ec17d6593f8/.github/workflows/release-embedding-sdk.yml#L425-L429